tools/csv-processor/csv-processor.go

```diff
@@ -157,9 +159,33 @@ Performance Addon Operator provides the ability to enable advanced node performa
 			TargetPort:              &targetPort,
 			SideEffects:             &sideEffects,
 			AdmissionReviewVersions: []string{"v1", "v1alpha1"},
-			WebhookPath:             &webhookPath,
+			WebhookPath:             &conversionWebhookPath,
 			ConversionCRDs:          []string{"performanceprofiles.performance.openshift.io"},
 		},
+		{
+			GenerateName:            "vperformanceprofile.kb.io",
+			Type:                    csvv1.ValidatingAdmissionWebhook,
+			DeploymentName:          strategySpec.DeploymentSpecs[0].Name,
+			ContainerPort:           443,
+			TargetPort:              &targetPort,
+			SideEffects:             &sideEffects,
+			AdmissionReviewVersions: []string{"v1beta1"},
```

Comment:
I unsure how does it work for the conversion webhook probably it does not return objects of the AdmissionReview at all, but v1 does not work because of the issue https://github.com/kubernetes-sigs/controller-runtime/issues/1161

Reply:
hmm, so maybe add both?

Reply:
one of them should work

Reply:
I unsure we should add things that will not work, once the controller-runtime will support v1 we can update it.

Reply:
as long as we remember :)